BTCPayServer/wwwroot/main/site.css

```
@@ -267,6 +279,291 @@ h2 .icon.icon-info {
+    padding: var(--btcpay-space-m) var(--btcpay-space-l);
+    background: none;
+    border: 0;
+    color: var(--btcpay-body-text-muted);
+    font-size: var(--btcpay-font-size-m);
+    font-weight: var(--btcpay-font-weight-semibold);
+    text-transform: uppercase;
+    letter-spacing: .08em;
+}
+.inv-collapse-head .icon {
+    --icon-size: 1.5rem;
+    transition: transform .2s ease;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)